ui.go

```diff
@@ -43,15 +43,32 @@ func newUI(w fyne.Window, p fyne.Preferences, host func() string) *ui {
 }
 
 func (u *ui) askShutdown() {
-	dialog.ShowConfirm("Shutdown", "Are you sure you want to shut down?",
-		func(ok bool) {
-			if !ok {
-				return
-			}
-
-			cmd := exec.Command("shutdown", "-h", "now")
-			_ = cmd.Start()
-		}, u.win)
+	var pop *widget.PopUp
+	message := widget.NewLabel("Are you sure you want to power off your computer?")
+	message.Alignment = fyne.TextAlignCenter
+
+	buttons := container.NewGridWithColumns(3,
+		widget.NewButtonWithIcon("Cancel", theme.CancelIcon(), func() {
+			pop.Hide()
+		}),
```

Comment:
```suggestion
		widget.NewButtonWithIcon("Cancel", theme.CancelIcon(), pop.Hide),
```
This can be Done without wrapping `Hide()` in an anonymous function.